packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageModule.kt

```diff
@@ -29,16 +32,24 @@ class ExpoImageModule : Module() {
   override fun definition() = ModuleDefinition {
     Name("ExpoImage")
 
-    AsyncFunction("prefetch") { urls: List<String>, cachePolicy: CachePolicy, promise: Promise ->
+    AsyncFunction("prefetch") { urls: List<String>, cachePolicy: CachePolicy, headersMap: ReadableMap?, promise: Promise ->
       val context = appContext.reactContext ?: return@AsyncFunction false
 
       var imagesLoaded = 0
       var failed = false
 
+      val headers = headersMap?.toHashMap()?.let {
```

Comment:
```suggestion
      val headers = headersMap?.let {
```
That won't be needed after changing the type.